nxcomp/src/Loop.cpp

```diff
@@ -2847,7 +2845,7 @@ int NXTransSelect(int *resultFDs, int *errorFDs, int *setFDs, fd_set *readSet,
 
   #ifdef TIME
 
-  diffTs = diffTimestamp(lastTs, getNewTimestamp());
```

Comment:
Good idea, but wrong implementation. You're now shadowing `diffTs` twice in `NXTransSelect`.

Declare it once at the start of the function (and be careful to *not* do it in the `#ifdef TIME` section, since it's also used once outside of any `#ifdef TIME` section) and just use it normally afterwards.